### ISSUE TITLE
Foundation.toggler.js does not update aria-controls with ids of multiple toggled nodes

### ISSUE THREAD

**Issue by MoarCoding** [zurb/foundation-sites#11166]
``` javascript
// Add ARIA attributes to triggers
    var id = this.$element[0].id;
    $(`[data-open="${id}"], [data-close="${id}"], [data-toggle="${id}"]`)
      .attr('aria-controls', id);
```

The current code only checks for a single id. I suggest updating to
``` javascript
// Add ARIA attributes to triggers
    var id = this.$element[0].id,
        $triggers = $(`[data-open~="${id}"], [data-close~="${id}"], [data-toggle~="${id}"]`);

    $triggers.each(function(index, element) {
       var $this = $(element);

       //make sure not to duplicate id in aria-controls
       $this.attr('aria-controls', `${$this.attr('aria-controls') ? $this.attr('aria-controls').replace(id, '') : ''} ${id}`.trim());
    }); 
``` 

#### How to reproduce this bug:
Compare https://foundation.zurb.com/sites/docs/toggler.html#toggle-a-css-class and https://foundation.zurb.com/sites/docs/toggler.html#multiple-targets

#### What should happen:
Toggler should get aria-controls

#### What happened instead:
Toggler only gets aria-controls when toggling a single node

**Comment by DanielRuf** [comment]
https://www.w3.org/TR/wai-aria-1.1/#aria-controls

Should be a [ID reference list](https://www.w3.org/TR/wai-aria-1.1/#valuetype_idref_list)

**Comment by DanielRuf** [comment]
Hi @MoarCoding,

thanks for reporting this.

You are right, `aria-controls` should contain all of them.

Do you want to provide a PR?

**Comment by MoarCoding** [comment]
https://github.com/zurb/foundation-sites/pull/11167
